.gitignore

```diff
@@ -1,3 +1,5 @@
 # Need a config/environment.rb for rails.vim to work in engines
 config/environment.rb
 *gem
+.idea
+.bundle
```

Comment:
Out of habit I always had .bundle since the rest of my repos need it.  I can remove it.